LoopTests/Extensions/DosingDecisionStoreTests.swift

```diff
@@ -10,935 +10,936 @@ import XCTest
 import HealthKit
 
 @testable import Loop
-@testable import LoopKit
+// LoopKit is not compiled for Testing in carthage builds. LoopKit private units should be tested in LoopKit
+//@testable import LoopKit
```

Comment:
This is a lot of unit tests being removed.  Were they moved to LoopKit, then?  If not, I have a concern over the reduction in unit test coverage.

Reply:
Yeah, the `@testable import LoopKit` should've never made it into Loop; I missed this in review, and will look into moving or redoing these tests so they don't depend on LoopKit internals.

Reply:
Found a way to keep the tests and not depend on LoopKit internals (we can't use LoopKit internals in tests when building non-workspace ways, like Carthage or SPM); had to expose a cleanup method in LoopKit, which is in the associated LoopKit PR. One test was removed, but the functionality exercised in it was covered with other tests.